test/integrant/test_runner.cljs

```diff
@@ -0,0 +1,6 @@
+(ns integrant.test-runner
+  (:require [doo.runner :refer-macros [doo-tests]]
+            [integrant.core-test]))
+
+(doo-tests 'integrant.core-test)
+
```

Comment:
Extra whitespace at the bottom of file. Single newline is good, double newline is bad :)